src/WebJobs.Script/ExtensionBundle/ExtensionBundleManager.cs

```diff
@@ -23,23 +25,25 @@ public class ExtensionBundleManager : IExtensionBundleManager
     {
         private readonly IEnvironment _environment;
         private readonly ExtensionBundleOptions _options;
+        private readonly FunctionsHostingConfigOptions _configOption;
         private readonly ILogger _logger;
         private readonly string _cdnUri;
         private string _extensionBundleVersion;
 
-        public ExtensionBundleManager(ExtensionBundleOptions options, IEnvironment environment, ILoggerFactory loggerFactory)
+        public ExtensionBundleManager(ExtensionBundleOptions options, IEnvironment environment, ILoggerFactory loggerFactory, FunctionsHostingConfigOptions configOption)
         {
             _environment = environment ?? throw new ArgumentNullException(nameof(environment));
             _logger = loggerFactory.CreateLogger<ExtensionBundleManager>() ?? throw new ArgumentNullException(nameof(loggerFactory));
             _cdnUri = _environment.GetEnvironmentVariable(EnvironmentSettingNames.ExtensionBundleSourceUri) ?? ScriptConstants.ExtensionBundleDefaultSourceUri;
             _options = options ?? throw new ArgumentNullException(nameof(options));
+            _configOption = configOption;
```

Comment:
Do we also want to do this just to be safe ` ?? throw new ArgumentNullException(nameof(configOption));`